bufferflow_timedbinary.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"encoding/json"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type BufferflowTimedBinary struct {
+	Name   string
+	Port   string
+	Output chan []byte
+	Input  chan []byte
```

Comment:
These could be private, they're never accessed outside `BufferflowTimedBinary`'s functions.